src/components/navBar/navBar.module.scss

```diff
@@ -82,28 +83,33 @@ $theme-active: #87d870;
     .hamburgerIcon {
         display: none;
     }
+
     .mobile_menu > li {
         list-style: none;
     }
+
     .navLinks .mobile_menu {
         display: flex;
         flex-direction: row;
         align-items: center;
         gap: 1rem;
     }
+
     .active {
         color: $theme-active;
         text-decoration: underline;
         text-decoration-thickness: $thickness;
         text-underline-offset: $offset;
     }
 }
+
 @media (max-width: 970px) {
     .navBar {
         display: flex;
         flex-direction: row;
         flex-wrap: wrap;
         position: relative;
```

Comment:
Please place this media query at the bottom of the file

Reply:
It is at the bottom, please check